crates/astria-sequencer-relayer/src/keys.rs

```diff
@@ -1,92 +1,97 @@
+use std::{
+    fs::File,
+    path::Path,
+};
+
 use bech32::{
     self,
     ToBase32,
     Variant,
 };
-use ed25519_dalek::{
-    Keypair,
-    PublicKey,
-};
-use eyre::WrapErr as _;
-use sha2::{
-    Digest,
-    Sha256,
+use ed25519_dalek::Keypair;
+use eyre::{
+    bail,
+    ensure,
+    WrapErr as _,
 };
+use serde::Deserialize;
+use tracing::instrument;
 
-const ADDRESS_LENGTH: usize = 20;
-
-/// converts an encoded private key to a ed25519_dalek::Keypair
-pub fn private_key_bytes_to_keypair(private_key: &[u8]) -> eyre::Result<Keypair> {
-    let private_key = ed25519_dalek::SecretKey::from_bytes(&private_key[..32])
-        .wrap_err("failed reading secret key from bytes")?;
-    let public_key = PublicKey::from(&private_key);
-    Ok(Keypair {
-        secret: private_key,
-        public: public_key,
-    })
+#[derive(Debug)]
+pub(crate) struct Validator {
```

Comment:
Agree, I wanted to `git mv keys.rs validator.rs` or something like that, but it made reviewing things impossible (because everything was just green). I wonder if that has changed.

Reply:
yeah I'd rename this to `ValidatorFile` or something like that, or add a comment about where this came from